### PR TITLE
added Analysis.html suggestions

### PR DESCRIPTION
**Pull request by mattsecrest**

### CODE REVIEW

**Comment by mattsecrest** [comment]
@gowerc made some suggestions to the user guide and `apply_mcmc` for consideration.

File: R/apply_mcmc.R [hunk 1]
```diff
@@ -57,16 +57,16 @@
...
 #' - **`alpha[1]`** - The shape parameter for the trial's baseline distribution
 #'
 #' - **`alpha[2]`** - The shape parameter for the historical control's baseline distribution
 #'
-#' - **`beta[1]`** - The log-hazard ratio for the treatment effect. This is equivalent to
+#' - **`beta_trt`** - The log-hazard ratio for the treatment effect. This is equivalent to
 #' `log(HR_trt_cc)`
 #'
-#' - **`beta[x]`** - The log-hazard ratio for any other covariate provided to the model via
+#' - **`beta_x`** - The log-hazard ratio for any other covariate provided to the model via
```
Comment (gowerc) [inline R/apply_mcmc.R:69]:
```suggestion
#' - **`beta_<var>`** - The log-hazard ratio for any other covariate provided to the model via
```

File: R/apply_mcmc.R [hunk 2]
```diff
@@ -85,6 +85,13 @@ apply_mcmc <- function(dt, formula_cov, ...) {
     if (!"(Intercept)" %in% colnames(design_mat)) {
         stop("Covariate formula must contain the intercept term")
     }
+
+    if (ncol(design_mat)==1) {
+      stop("Right now, apply_mcmc is designed to incorporate conditional
+      models with all covariates of interest. The function cannot be run
+      without additional covariates (i.e.,  with formula_cov: ~ 1L)")
+    }
```
Comment (gowerc) [inline R/apply_mcmc.R:93]: I wasn't aware of this, does it actually fail if there are no covariates ?

File: vignettes/Analysis.Rmd [hunk 3]
```diff
@@ -47,56 +48,51 @@ $$
 with the rate taking the format:
 
 $$
-\lambda_i = exp(\alpha_i + \beta_{trt}  * I_{trt} + \beta_i * x_i)
+\lambda_i = exp(\alpha_i + \beta_{trt}  \times I_{trt} + \beta_n^T \times X_{ni})
```
Comment (gowerc) [inline vignettes/Analysis.Rmd:51]: Is the `_n` suffix needed ?

File: R/apply_mcmc.R (hunk 2 above)
Comment (gowerc) [inline R/apply_mcmc.R:93]:
```suggestion

    if (ncol(design_mat)==1) {
      stop("The function cannot be run without additional covariates (i.e. `formula_cov = ~ 1 ` is not supported)")
    }
```

File: vignettes/Analysis.Rmd (hunk 3 above)
Comment (gowerc) [inline vignettes/Analysis.Rmd:51]:
```suggestion
\lambda_i = exp(\alpha_i + \beta_{trt}  \times I_{trt} + \beta^T \times X_{i})
```

**Pull request merged**